_topics/en/eclipse-attacks.md

```diff
@@ -82,6 +82,12 @@ optech_mentions:
   - title: "Bitcoin Core 0.20 released with asmap feature to improve eclipse resistance"
     url: /en/newsletters/2020/06/10/#bitcoin-core-0-20-0
 
+  - title: "Bitcoin Core #19670 reserves inbound slots for block-relay-only peers"
+    url: /en/newsletters/2020/09/09/#bitcoin-core-19670
+
+  - title: "CVE-2018-17145 InvDoS attack could've been used to eclipse nodes"
```

Comment:
Is InvDoS common terminology? I don't see it on the mailing list post or in the linked paper.

Reply:
It's not in the paper, but it's in its URL: https://invdos.net/paper/CVE-2018-17145.pdf .  And that's what's in the logo:

![](https://invdos.net/assets/invdos.png)

Reply:
🤦‍♂️